packages/ods/src/components/text/documentation/specifications/specifications-text.mdx

```diff
@@ -1,7 +1,4 @@
 import { Markdown } from '@storybook/addon-docs';
 import Specs from './spec.md';
 
-## Description
```

Comment:
keep the title, remove just the TODO

Reply:
There are a duplicate information with this file: 
packages/storybook/stories/components/text/1_specifications.stories.mdx
That's why I remove it

Reply:
Careful there is two different part here
<img width="1035" alt="Capture d’écran 2024-03-18 à 17 45 15" src="https://github.com/ovh/design-system/assets/1390934/2139b973-9819-49de-9475-5d8a316cbdc5">

This description is a sub part of the `# Text - Technical Specification` from the file you mentionned